project.clj

```diff
@@ -1,6 +1,6 @@
 (def ks-version "1.0.0")
 
-(defproject puppetlabs/trapperkeeper "1.2.1-SNAPSHOT"
+(defproject puppetlabs/trapperkeeper "1.3.0-SNAPSHOT"
```

Comment:
Yeah, that makes sense.  Sorry, was thinking we'd already done a T/K release with the earlier form of HUP.  Losing track of things....